### ISSUE TITLE
support for roots/acorn and other framworks that dont implement thw whole of laravel/framework

### ISSUE THREAD

**Issue by chrillep** [gehrisandro/tailwind-merge-laravel#12]
Background:
So there are multiple ways of using all the good stuff laravel provides.
One of them is [acorn](https://github.com/roots/acorn). But it does not depend on the whole of laravel but instead of the specific pieces.
So in order to be compatible with other implementations 

use required illuminate packages instead of laravel/framework

https://github.com/gehrisandro/tailwind-merge-laravel/blob/091dec8499c5b8ad2268369a26a329af78eab73f/composer.json#L15

refs:
* https://discourse.roots.io/t/twmerge-in-sage-10/26631/10

**Comment by gehrisandro** [comment]
Hi @chrillep 

I would suggest using TailwindMerge for plain PHP https://github.com/gehrisandro/tailwind-merge-php

**Comment by chrillep** [comment]
> Hi @chrillep
> 
> I would suggest using TailwindMerge for plain PHP https://github.com/gehrisandro/tailwind-merge-php
yeah bu then i need to duplicate your provider since acorn supports that.
But i hear you
